### DISCUSSION TITLE
EKS version upgrades compatibility with Strimzi CRDs

### DISCUSSION THREAD

**Discussion by doverk96** [strimzi/strimzi-kafka-operator#6577] [Q&A]
Hey,
Any such compatibility link which can help me determine what all CRD definition and component definition I need to take care of when updating my EKS versions.
Have two cluster - at 1.17 and other at 1.19, will take both to 1.21.
What all I need to take care for a smooth Strimzi behaviour during update process and after it.

**Comment by scholzj** [comment]
You can find the oldest supported version on the website - on the downloads section: https://strimzi.io/downloads/. The oldest supported is a bit more tricky to determine since Kubernetes versions continue to be released even after some Strimzi version is released. In general, Strimzi versions up to 0.22 (including) are not compamtible with Kube 1.22 and newer. Strimzi 0.23 and newer should be compatible with the latest Kubernetes versions. It is always recommended to test the upgrade with some test environment first as every setup is a bit different and every Kubernetes environment does the upgrade a bit differently.
